test/e2e/deployment.go

```diff
@@ -613,24 +613,14 @@ func testRollbackDeployment(f *Framework) {
 	Expect(err).NotTo(HaveOccurred())
 	defer stopDeployment(c, f.Client, ns, deploymentName)
 
-	// Check that deployment is created fine.
-	deployment, err := c.Extensions().Deployments(ns).Get(deploymentName)
+	// Wait for DeploymentStatus to be appropriately updated
+	deploymentReplicasPtr := util.IntPtr(deploymentReplicas)
+	err = waitForDeploymentStatusUpdated(c, ns, deploymentName, deploymentReplicasPtr, deploymentReplicasPtr, deploymentReplicasPtr, util.IntPtr(0))
```

Comment:
Just in case the caller only wants to wait for some of the status being updated (say, they don't care about available pods) in the future?